github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -841,7 +841,6 @@ presubmits:
       preset-dind-enabled: "true"
       preset-docker-mirror-proxy: "true"
     name: pull-kubevirt-verify-go-mod
-    run_if_changed: ^vendor/.*|^staging/.*|go\.mod$|go\.sum$
```

Comment:
do you want always_run: true ? (line 831)

Reply:
Yup!
Done, thanks for catching that